src/lib/routes/helpers.ts

```diff
@@ -35,7 +35,7 @@ export function isStateAtTabRoot(state: State | undefined) {
   return (
     isTab(currentRoute.name, 'Home') ||
     isTab(currentRoute.name, 'Search') ||
-    isTab(currentRoute.name, 'Feeds') ||
+    isTab(currentRoute.name, 'Messages') ||
```

Comment:
When we switched out FeedsTab for MessagesTab, we missed it here. One thing this does is allow drawer swipe while on messages screen.